code/modules/clothing/suits/labcoat.dm

```diff
@@ -195,9 +195,10 @@
 	desc = "A sturdy labcoat to protect frontline reserchers"
 	icon_state = "armored_coat"
 	item_state = "armored_coat"
-	flags_armor_protection = CHEST|ARMS
-	soft_armor = list("melee" = 35, "bullet" = 30, "laser" = 20, "energy" = 30, "bomb" = 25, "bio" = 30, "rad" = 0, "fire" = 20, "acid" = 30)
-	slowdown = 0.3
+	flags_armor_protection = CHEST|GROIN|ARMS|LEGS
+	light_range = 8
```

Comment:
Ignore this, i thought i removed it, has no effect on anything.

Reply:
```suggestion
```